src/plugins/ui/evaluation_conditional_format.ts

```diff
@@ -380,7 +376,7 @@ export class EvaluationConditionalFormatPlugin extends UIPlugin {
       if (cell && cell.evaluated.type === CellValueType.error) {
         return false;
       }
-      const values = rule.values.map(parsePrimitiveContent);
+      const values = rule.values?.map(parsePrimitiveContent) || [];
```

Comment:
is this necessary ? The type definition states that `values` is always defined. But maybe it's lying :p

Reply:
the data fed to `Model` unfortunately bypasses it :/. I feel like TS could warn me that the chaining `?.` is unnecessary since values is always defined 🤔